level1/f256/modules/vtio.asm

```diff
@@ -367,7 +408,10 @@ Init                stu       D.KbdSta
                     
                     lbsr      InitDisplay         initialize the display
                     lbsr      InitSound           initialize the sound
-                    lbsr      InitKeyboard        initialize the keyboad                    
+                    lbsr      InitKeyboard        initialize the keyboad
+		    ifgt      Level-1
```

Comment:
Why is this conditional here if we support Level 1?

Reply:
Removed support for Level-1 to remove from FEU. Since FEU uses L1 vtio, it was easier to strip the mouse from L1 than to make a conditional for the FEU.